README.md

```diff
@@ -1,488 +1,742 @@
-[![Build Status](https://travis-ci.org/partiql/partiql-ir-generator.svg?branch=master)](https://travis-ci.org/partiql/partiql-ir-generator)
+Travis: [![Build Status](https://travis-ci.org/partiql/partiql-ir-generator.svg?branch=master)](https://travis-ci.org/partiql/partiql-ir-generator)
+Generator: [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.partiql/partiql-ir-generator/badge.svg?)](https://search.maven.org/artifact/org.partiql/partiql-ir-generator)
+Runtime: [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.partiql/partiql-ir-generator-runtime/badge.svg?)](https://search.maven.org/artifact/org.partiql/partiql-ir-generator-runtime)
 
 # The PartiQL I.R. Generator
 
-PIG is a compiler framework, domain modeling tool and code generator for tree data structures such as ASTs (Abstract Syntax Tree), database logical plans, database physical plans, and other intermediate representations.  Using PIG, the developer concisely defines the structure of a tree by specifying named constraints for every node and their attribues. Every constraint is known as a "data type", a collection of data types is known as a "type domain" and a collection of type domains is known as a "type universe".
+- [The PartiQL I.R. Generator](#the-partiql-ir-generator)
+    * [PIG Overview](#pig-overview)
```

Comment:
As a side note, creating this table of contents may not be necessary. GitHub can create a list of headings by default.
![image](https://user-images.githubusercontent.com/23245405/150875977-de22d412-31c5-4e69-940d-5919a3df5362.png)

Reply:
I'll have to look at that.  Perhaps it is a relatively new feature.

Reply:
So it is not truly needed.  Thanks.  No idea how I never noticed that before.